lib/matplotlib/axes/_axes.py

```diff
@@ -5400,10 +5414,12 @@ def pcolor(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
 
             pcolor(C.T)
 
-        MATLAB :func:`pcolor` always discards the last row and column of ``C``,
-        but Matplotlib displays the last row and column if ``X`` and ``Y`` are
-        not specified, or if ``X`` and ``Y`` have one more row and column than
-        ``C``.
+        **Difference to MATLAB pcolor()**
```

Comment:
again:

**Handling of pcolor end-cases**

`pcolor` displays all columns of *C* if *X* and *Y* are not specified, or if *X* and *Y* have one more column than *C*.  If *X* and *Y* have the same number of columns as *C* then the last column of *C* is dropped.   Similarly for the rows.